### DISCUSSION TITLE
Manifest.json?

### DISCUSSION THREAD

**Discussion by tony** [pradyunsg/furo#479] [Ideas] [answered]
Do any solutions exist in furo, or its [recommendations](https://pradyunsg.me/furo/recommendations/) that gives [manifest.json](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json)?  Through searching, it seems there isn't much discussion of manifest.json in sphinxdoc at all.

When using https://cards-dev.twitter.com/validator it seems that none of the sphinx websites I see have card previews.

**Comment by pradyunsg** [comment, marked as answer]
Not in Furo or its current recommendations, no. If you do find something that provides this, please lemme know and I'll probably add it to the recommendations page. :)

FWIW, cards can be generated using sphinx-opengraph which is mentioned in the docs.
